packages/dds/ink/src/ink.ts

```diff
@@ -99,6 +99,7 @@ const snapshotFileName = "header";
  * ink.on("clear", this.renderClear.bind(this));
  * ```
  * @sealed
+ * @public
```

Comment:
i would probably mark this whole dds as beta. i don't think it is well used or tested. 
curious what @ChumpChief  thinks.

Reply:
We already ship these as public. I don't think we can mark them as beta without going through a deprecation cycle.

Reply:
Annotating them as `alpha` or `beta` will eventually mean that the items disappear from our standard APIs. We plan on moving to a world where only `public` items are available by default, and consumers will need to explicitly opt in to using experimental APIs.

I'm curious about potentially moving this to experimental as well, since I don't think this has active use / I don't think we're actively maintaining it? @anthony-murphy maybe you can confirm?

Either way, I think such changes are out of the scope of this PR, but I would love to develop a plan.

Reply:
I'd go alpha and/or experimental.  No customers currently and will likely need to undergo some serious breaking changes to become valuable for a customer (if we decide to invest in it).

Reply:
This package was established prior to our current experimental/alpha conventions, but this experimental status is reflected/communicated in TODO.txt:

> Ink is an in-progress data structure with the purpose of facilitating collaborative inking. There is a set of anticipated work to be done still, including breaking changes across Ink and the Canvas example. Please do try it out and let us know what you think, but also be prepared for the following incoming changes...

Reply:
Is there opposition to removing the ink dds, rather than marking it `@alpha`?

Reply:
I'd like to keep it as alpha.

Reply:
Any particular reason? It seems this DDS is unmaintained and we don't have any plans currently to maintain or invest in it further. The code would still live in the git history.

Reply:
I would second @connorskees's question / argument for removing it. Unless we gain some API/test coverage by keeping it alive, we are paying a maintenance task for no real benefit. If it's beneficial for test coverage, then I would argue we need to add better unit test coverage elsewhere. And I think we have a sufficient number of other DDSs, that I'm not sure this one is adding much in terms of API validation.

That said, if we _do_ want to keep it (or if we want to keep it alive for now while we continue to discuss the prospect of removing it), I have a draft PR out that moves it to the "experimental" directory and marks its APIs as `@alpha`: https://github.com/microsoft/FluidFramework/pull/18145.